sources/api/datastore/src/serialization/pairs.rs

```diff
@@ -540,13 +541,14 @@ mod test {
             boolean: true,
         };
         let a = A { id: 42, b: Some(b) };
-        let keys = to_pairs(&a).unwrap();
+        let j = serde_json::to_value(a).unwrap();
```

Comment:
nit: could be a good idea to define these test structures using the `json!` macro, but it isn't necessary.